### PR TITLE
Snakefile pathfix

### PR DESCRIPTION
**Pull request by pontushojer**
Related to #25 

- Moved Snakefile and included in package so that it could be located by setuptools. Also included a empty \_\_init\_\_.py file in the dbspro folder that was needed for this. 
- Removed config.yaml file. TSV files containing ABC and handle info will instead be given as arguments to run.py (or default ones used from construct-info folder).

### CODE REVIEW

**Comment by pontushojer** [comment]
So apparently all is not good, will work on a fix-fix.

File: src/dbspro/cli/run.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 accepted_file = 'reads.fastq.gz'
 accepted_file_ext = '.fastq.gz'
 
+default_handles_file = "construct-info/handles.tsv"
+default_abc_file = "construct-info/ABC-sequences.tsv"
```
Comment (FrickTobias) [inline src/dbspro/cli/run.py]: I think these could be written in the `default` in the argparser.

Comment (FrickTobias) [reply src/dbspro/cli/run.py]: That also means lines 53-57 are not needed.

**Comment by pontushojer** [comment]
So now everything is working again. Major issues with handling default paths on travis so I decided not to and instead decided to require that handles and ABC-sequences have to be give explicitly in the run command i.e. 

```
dbspro run -d outdir -f testdata/reads-10k-DBS.fastq.gz --handles-file construct info/handles.tsv --abc-file construct-info/ABC-sequences.tsv
```
You can start the review now @FrickTobias

File: src/dbspro/cli/run.py (hunk 1 above)
Comment (pontushojer) [reply src/dbspro/cli/run.py]: Yes they are removed in last version

**Pull request merged**